xunit.runner.wpf/ViewModel/TraitCollectionView.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Collections.ObjectModel;
+using System.Collections.Specialized;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace xunit.runner.wpf.ViewModel
+{
+    public sealed partial class TraitCollectionView
+    {
+        private readonly TraitViewModelComparer _comparer = TraitViewModelComparer.Instance;
+        private readonly ObservableCollection<TraitViewModel> _collection = new ObservableCollection<TraitViewModel>();
+
+        public ObservableCollection<TraitViewModel> Collection => _collection;
+
+        public TraitCollectionView()
+        {
+
+        }
+
+        public void Add(ImmutableArray<TraitViewModel> traitList)
+        {
+            if (traitList.Length == 0)
```

Comment:
Is this necessary to protect against getting `default(ImmutableArray<TraitViewModel>)`?  If so, perhaps a more explicit check?